pom.xml

```diff
@@ -12,6 +12,7 @@
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <dep.jackson.version>2.8.10</dep.jackson.version>
```

Comment:
I'm super into this change generally, but out of curiosity why `2.8.10`over a newer version?

Reply:
JJSchema currently depends on jackson 2.22. Downgrading to 2.8 doesn't make sense. Could you try a newer version of jackson?

Reply:
jersey 2.22, not jackson. and the version you pull in in is some randomly packaged together thing from eclipsesource that is not even the latest (2.26). I don't think that there is a big problem, most people either do not care or exclude that dependency, which is messy). 

 I chose the version of jackson somewhat at random (matching my project, which uses dropwizard 1.1.x) but there is no reason why it could not be 2.9.3 (the latest).

Reply:
Oh my! I'm seeing things! 🙈 

Sure... 2.8 is already good.